plugin/build.gradle

```diff
@@ -207,7 +207,8 @@ jacocoTestReport {
 List<String> jacocoExclusions = [
         // TODO: add more unit test to meet the minimal test coverage.
         'org.opensearch.ml.constant.CommonValue',
-        'org.opensearch.ml.plugin.MachineLearningPlugin*'
+        'org.opensearch.ml.plugin.MachineLearningPlugin*',
+        'org.opensearch.ml.task.MLPredictTaskRunner'
```

Comment:
We might want to refactor this PredictTaskRunner. It has a big body in predict() method which is not test friendly.